anta/cli/debug/utils.py

```diff
@@ -54,8 +55,8 @@ def wrapper(
         try:
             d = inventory[device]
         except KeyError as e:
-            message = f"Device {device} does not exist in Inventory"
-            logger.error(e, message)
+            message = f"Device '{device}' does not exist in Inventory"
+            anta_log_exception(e, message)
```

Comment:
Not really what we want.
`anta_log_exception` print a traceback to debug.
This except block does not need to be debug.
It is more something like
```
if device not in inventory:
   logger.error(f"Device {device} does not exist in Inventory")
   ctx.exit(ExitCode.USAGE_ERROR)
```

That we need.

Reply:
`anta_log_exception` does not `print a traceback to debug`. It adds a log for the exception either in `critical` or if ANTA_DEBUG is set, it prints the traceback (which granted we don't need in this case)
https://github.com/arista-netdevops-community/anta/blob/6b7a54ff93c76250d242d3796f6cc84bb51813e0/anta/logger.py#L116 (critical if no ANTA_DEBUG set, exception otherwise)

Reply:
ok that was not the point - we should not raise ;) coffee needs more ☕